src/codegen_call_dispatch.cpp

```
@@ -137,8 +137,14 @@ llvm::Value *CodeGen::emitExprVariant(const std::unique_ptr<CallExpr> &e) {
             std::vector<llvm::Value*> argVals;
             for (auto &arg : e->args)
                 argVals.push_back(emitExpr(*arg));
+
+            auto ucTemps = wrapFnTypedArgs(argVals, info.paramTypeNames);
+
             llvm::Value *loaded = builder_.CreateLoad(ptrTy_, varPtr, e->callee + ".fn");
```

Clang (14.0.6)

[warning] variable 'loaded' is not initialized

(cppcoreguidelines-init-variables)